src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -410,6 +410,7 @@ def __init__(
         env_vars=None,
         workload_profile_name=None,
         ingress=None,
+        force_single_container_updates=None
```

Comment:
You add a new argument `force_single_container_updates`?
Need to update `_params.py` and add description about it.

Reply:
Also, please add an example for the new argument in `_help.py`.